### DISCUSSION TITLE
VM / network - disabled component ?

### DISCUSSION THREAD

**Discussion by nicopilami** [monarc-project/MonarcAppFO#429] [Q&A] [answered]
Hi
I downloaded the Monarc VM to install it on my Synology NAS.

I'm not familiar with Linux , thus, please, could someone tell me how to enable the ethernet card of the ubuntu VM, as it seems it has its ethernet "disabled" so that I can't get the network packages from the internet (ie; network-manager, net-tools) ?

(I already looked at this page, without success : https://www.monarc.lu/documentation/common-issues/)

Thanks a lot.
Nico

**Comment by ruslanbaidan** [comment]
Hello,
The Virtual Machine is not used to be installed on a physical or virtual servers, but to import and use on VirtualBox. Technically it's possible but not recommended in any case.
Please use the following Installation instructions to preform the process depending on your Linux OS distribution.
https://github.com/monarc-project/MonarcAppFO/tree/master/INSTALL

**Reply by nicopilami** [reply]
Hi thanks for your answer.
Actually this OVA has been well imported into the Virtual Machine management component of the NAS; 
The only issue I can see here is that the network adapter seems disabled.
this does not come from the NAS as other VMs are running well :)

![image](https://user-images.githubusercontent.com/6035168/165304879-c35ca3e7-aa42-4f4d-b825-1d1e80f97b85.png)

**Comment by ruslanbaidan** [comment]
The VM is built with use Packer.
You can try to configure your network adapter before launching the VM or restart it. The IP address of the server can be the access point of the app from http protocol in browser.
You may need to configure https certificate to work with apache.

**Reply by nicopilami** [reply]
not sure I know how to do this, but I'll use the RTFM method :D

Actually OK seems ok on the hypervisor running on the NAS, so that I guess the issue comes from the VM when it has been packed with the adapter disabled, but... maybe i'm wrong. 

Anyway I confirm Monarc works great while being hosted locally on my workstation's harddrive, however the displayed IP adress which is wrong, at least on my own setup. (the system can be accesses by navigating to http://...)

I'll try what you advice and if it still does not work, or if I do not find "how to" perform the actions, then i'll try to setup a new debian VM and install Monarc "from scratch" on it. 

Thanks for your (quick) support, I'll come back here very soon !

**Comment by ruslanbaidan** [comment, marked as answer]
It would be better as you will have a full control of the application, that is very important: performing updates, easy db backups, control and install Stats Service, BackOffice if needed.
You might consider to use Ubuntu 18.04. Here are the installation instructions: https://github.com/monarc-project/MonarcAppFO/blob/master/INSTALL/INSTALL.ubuntu1804.md
VM is good for testing purposes or can be used for a single user.

**Reply by nicopilami** [reply]
just downloading Ubuntu 20.04LTS :)

**Reply by K8Sudo** [reply]
for 20.04 this can help https://github.com/KonzeptAcht/monarc-installation-instructions/blob/main/EN/ubuntu-20.04-en.md
